proto/Cargo.toml

```diff
@@ -19,10 +19,11 @@ bytes = "0.4"
 prost = "0.5"
 prost-derive = "0.5"
 protobuf = "~2.2"
+lazy_static = "1.3"
 
 [build-dependencies]
 grpcio-compiler = { path = "../compiler" }
-protobuf-build = "0.2"
+protobuf-build = "0.6.2"
```

Comment:
Better to just use "0.6" here. Then we'll get more improvements without having to land changes.

Reply:
I try "0.6" before, but got some warnings. Hmmm... you told me that 0.6.2 fixed this problem.

Reply:
Version 0.6.2 is the version of the crate with the fix. Using "0.6" means Cargo will choose from 0.6.0, 0.6.1, 0.6.2, and in the future, 0.6.3, etc. If you use just "0.6" it should be fine, you might need to `cargo update -p protobuf-build`, but I don't think you'll need to.

Reply:
understood, fixed

Reply:
Choosing 0.6.2  should mean any version that is >=0.6.2 && < 0.7.0 , so it can still get further improvements.